packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -393,7 +393,7 @@ class CupertinoTextField extends StatefulWidget {
     this.onSubmitted,
     this.onTapOutside,
     this.inputFormatters,
-    this.enabled,
+    this.enabled = true,
```

Comment:
I wonder if we should just make this a private variable inside of `CupertinoTextFieldState` similar to how `TextFieldState` does it. I'm not too opinionated on this either way but it would keep the API consistent between both `TextField`s.

Reply:
`TextField.enabled?` is a `bool?` because it can default to `InputDecoration.enabled` I think. Here it doesn't have anything to default to other than the const `true` so I don't know if there's a good reason to keep this `bool?`. 

I don't think we need to store `enabled` in CupertinoTextFieldState since the configuration is completely determined by the widget itself (`CupertinoTextField.enabled`), unless I'm missing something?

Reply:
I didn't consider the defaults in `InputDecoration`, that makes sense to me then.